Tests/FluentAssertions.Specs/Specialized/ExecutionTimeAssertionsSpecs.cs

```diff
@@ -615,6 +615,19 @@ public void When_asserting_execution_time_of_null_it_should_throw()
             act.Should().ThrowExactly<ArgumentNullException>()
                 .WithParameterName("action");
         }
+
+        [Fact]
+        public void When_accidentally_using_equals_it_should_throw_a_helpful_error()
+        {
+            // Arrange
+            var subject = new object();
+
+            // Act
+            Action act = () => subject.ExecutionTimeOf(s => s.ToString()).Should().Equals(1.Seconds());
+
+            // Assert
+            act.Should().Throw<NotSupportedException>().WithMessage("Equals is not part of Fluent Assertions. Did you mean BeLessThanOrEqualTo() or BeGreaterThanOrEqualTo() instead?");
```

Comment:
There is no `Be` as this would be likely to generate flaky tests, but since they accidentally put `Equals`, I figured it made sense to recommend `BeLessThanOrEqualTo` and its friend versus `BeLessThan`.  No worries to change if you prefer the other.